### DISCUSSION TITLE
SHP zeigt plötzlich mansche Werte nicht mehr an

### DISCUSSION THREAD

**Discussion by woupi** [camueller/SmartApplianceEnabler#288] [Installation und Betrieb]
Ich habe insgesamt 14 Zähler und mindestens einer wird nicht mehr angezeigt.
Hier das SEMP Protokoll für das betreffende Gerät:
`<DeviceStatus>
<DeviceId>F-23051969-000000000007-00</DeviceId>
<EMSignalsAccepted>false</EMSignalsAccepted>
<Status>On</Status>
<PowerConsumption>
<PowerInfo>
<AveragePower>77</AveragePower>
<Timestamp>0</Timestamp>
<AveragingInterval>60</AveragingInterval>
</PowerInfo>
</PowerConsumption>
</DeviceStatus>`

Sieht für mich i.O. aus.

Dieser Zähler ist für die Waschmaschine, diese zeigt im SHP 0W an
![Waschmaschine](https://user-images.githubusercontent.com/98451301/176372393-1e6f4610-77b2-45fd-9e02-91b0a754da14.jpg)

**Comment by camueller** [comment]
Perfekt analysiert: An der SEMP-Schnittstelle werden 77W und Schaltstatus=On übergeben, aber das Sunny Portal zeigt die Leistung nicht an. Liegt eindeutig am Sunny Portal. Möglicherweise löst sich das Problem von selbst, ansonsten Fehler melden bei SMA.

**Comment by woupi** [comment]
Danke für die schnelle Antwort und so hatte ich mir das auch schon gedacht, bin aber froh, dass du es noch schnelle bestätigt hast.
